### PR TITLE
docs: Update README.md

### PR DESCRIPTION
**Pull request by Murtatrxx**
It’s better IG

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,19 +15,15 @@
 </div>
 
 ## Why?
-- Most handlers don't support discord.js 14.7+
-- Customizable, composable commands
+- Featherweight, lightning-fast. A framework that's tailored to your exact needs.
```
Comment (jacoobes) [inline README.md:18]: Havent proven that its lightning fast, cant falsely advertise!

Comment (jacoobes) [reply README.md:18]: Not feather weight either its smaller than sapphire but not super light

File: README.md [hunk 2]
```diff
@@ -15,19 +15,15 @@
...
+- Support for discord.js v14 and all of it's interactions
+- Hybrid, customizable, composable commands. Just how you like.
 - Plug and play or customize to your liking
-- Embraces reactive programming for consistent and reliable backend
+- Developed reactive. For consistent and reliable backend
```
Comment (jacoobes) [inline README.md:22]: Incorrect grammar, revert

File: README.md [hunk 3]
```diff
@@ -15,19 +15,15 @@
...
-## 👀 Quick Look
-
-* Support for discord.js v14 and all interactions
-* Hybrid commands
-* Lightweight and customizable
-* ESM, CommonJS and TypeScript support
-* A powerful CLI and awesome community-made plugins
+- We're active and growing, always here to help. [Join the community](https://sern.dev/discord)
+- Use it with TypeScript, ESM, or CommonJS. All welcome.
+- Unleash the full potential with a poweful CLI and awesome plugins.
```
Comment (jacoobes) [inline README.md:26]: poweful -> powerful

**Review by jacoobes — CHANGES_REQUESTED** [review]

**Comment by jacoobes** [comment]
I don't really like these changes. The original made sense and the rewording makes it convoluting in my opinion . You add typos and don't bother to check before requesting a review

**Comment by Murtatrxx** [comment]
Changes makes it pop more

**Comment by Murtatrxx** [comment]
Why there's two section doing absolutely the same thing?

**Pull request closed without merging**